src/routes/roles/handlers.js

```diff
@@ -2,6 +2,7 @@ import { getAllRoles } from '../../services/roles';
 
 /* eslint-disable import/prefer-default-export */
 export async function allRoles(req, res) {
-  const topics = await getAllRoles();
+  const { onlySpecialist } = req.query;
+  const topics = await getAllRoles(onlySpecialist);
```

Comment:
I would have thought "false" as a URLParam evaluate to a string and not a boolean. 🤷

Reply:
That looks to be the case, either true or false on the Goal Form filters out only specialists. I don't object to a query param but this might be a case where an additional route _could_ be slightly neater to implement 
```/api/roles/``` & ```/api/roles/specialists/```

Your call though